File: app/src/main/java/org/example/gameByBall2/Ex_B.java
package org.example.gameByBall2;

import java.util.Arrays;

import processing.core.PApplet;

public class Ex_B {
  public static class Main extends PApplet {

    public void settings() {
      size(1000, 100);
      noLoop();
    }

    public void draw() {
      var array = new int[10];
      var rad = Arrays.stream(array).map(n -> (int) random(10, 100)).toArray();
      int count = 0;
      for (var r : rad) {
        ellipse(count * 100 + 50, 50, r, r);
        count++;
      }
    }
  }

  public static void main() {
    PApplet.main(Main.class.getName());
  }
}

File: app/src/main/kotlin/org/example/gameByBall2/Ex_B.kt
package org.example.gameByBall2

import processing.core.PApplet

object ExB {
    class Main : PApplet() {
        override fun settings() {
            size(1000, 100)
            noLoop()
        }

        override fun draw() {
            val array = IntArray(10)
            val rad = array.map { random(10f, 100f).toInt() }
            for ((i, r) in rad.withIndex()) {
                ellipse(i * 100f + 50f, 50f, r.toFloat(), r.toFloat())
            }
        }
    }

    fun main() {
        PApplet.main(Main::class.java.name)
    }
}
